packages/d3fc-series/README.md

```diff
@@ -120,6 +121,34 @@ line(data);
 Because D3 data-joins and data-binding only work on HTML / SVG, the canvas components are invoked directly with the supplied data. This causes the component to render itself to the canvas.
 
 
+#### WebGL rendering
+
+***WebGL rendering is not feature complete. Please check the API documentation against the methods to see what is currently supported.***
```

Comment:
It might be worth making this sound a bit more optimistic, there are certainly enough features supported to make a whole host of typical visualisations entirely possible.